src/controller/java/zap-generated/chip/devicecontroller/ClusterInfoMapping.java

```diff
@@ -14634,4 +14636,1716 @@ public void combineCommand(
     readAttributeMap.put("windowCovering", readWindowCoveringInteractionInfo);
     return readAttributeMap;
   }
+
+  public Map<String, Map<String, InteractionInfo>> getWriteAttributeMap() {
+    Map<String, Map<String, InteractionInfo>> writeAttributeMap = new HashMap<>();
+    Map<String, InteractionInfo> writeAccountLoginInteractionInfo = new LinkedHashMap<>();
+    // write attribute
```

Comment:
Try not to add comments in block iterators, they add a lot of lines in the generated code for no reason. You should remove `// read attribute` as well.